src/style.css

```diff
@@ -0,0 +1,43 @@
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) center no-repeat;
+  background-size: center;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.stars--2 :nth-child(-n+2) {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.stars--3 :nth-child(-n+3) {
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.stars--4 :nth-child(-n+4) {
+
+  background: url(images/star-active.svg) center no-repeat;
+}
+
+.stars--5 :nth-child(-n+5) {
+  background: url(images/star-active.svg) center no-repeat;
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/96409563/a9b68bf8-5719-40e9-827a-bdc8b696d039)
Also, you can change only background-image property. All other properties remain the same